src/main/java/co/org/osso/wallet/AccountRepository.java

```diff
@@ -0,0 +1,6 @@
+package co.org.osso.wallet;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface AccountRepository extends CrudRepository<Account, Integer> {
```

Comment:
Nice! You found the solution I hoped you would. I think it's more typical to use `Long` as the type of the primary key. Unless you have reasons to use `Integer`, we should probably do that instead.